homeassistant/components/tuya/base.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+"""Tuya Home Assistant Base Device Model."""
+from __future__ import annotations
+
+from tuya_iot import TuyaDevice, TuyaDeviceManager
+
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class TuyaHaDevice(Entity):
```

Comment:
To avoid confusion, let's call the entity base class Entity and not Device.

```suggestion
class TuyaHaEntity(Entity):
```

Reply:
fixed